src/App.js

```diff
@@ -3,88 +3,52 @@ import { Route, Switch } from 'react-router-dom'
 import styled from 'styled-components/macro'
 import useFetch from '../src/hooks/useFetch'
 import Categories from './components/Categories/Categories'
-import Detailpage from './components/Detailpage/Detailpage'
+import ProductDetail from './components/ProductDetail/ProductDetail'
 import ProductList from './components/ProductList/ProductList'
-import { ReactComponent as Logo } from './images/icons/logo.svg'
+import { ReactComponent as Logo } from './images/icons/planternate-logo.svg'
+import { categories } from './services/categories'
 
 export default function App() {
-  const burgers = useFetch('burger')
-  const kebabs = useFetch('kebab')
-  const minced = useFetch('minced')
-  const nuggets = useFetch('nuggets')
-  const sausages = useFetch('sausages')
-  const categories = useFetch('categories')
+  const products = {
+    burger: useFetch('burger'),
+    kebab: useFetch('kebab'),
+    minced: useFetch('minced'),
+    nuggets: useFetch('nuggets'),
+    sausages: useFetch('sausages')
+  }
 
   const [favorites, setFavorites] = useState(loadLocally('favorites') ?? [])
-  
+
   useEffect(() => {
     saveLocally('favorites', favorites)
   }, [favorites])
 
-
   return (
     <Switch>
       <Route exact path="/">
         <LogoStyled />
-        {categories.map((category) => (
-          <Categories
-            key={category.name}
-            productIcon={category.icon}
-            productName={category.name}
-          />
+        {categories.map(({name, icon}) => (
+          <Categories key={name} categoryIcon={icon} categoryName={name}/>
```

Comment:
Couldn`t name be tricky when you add more products? Do have access to any ID?

Reply:
Good call! I'm gonna change that